apps/api/src/validation/import-schemas.ts

```
@@ -0,0 +1,112 @@
+
+export const importHistoryQuerySchema = Joi.object({
+  page: Joi.number().integer().min(1).default(1),
+  pageSize: Joi.number().integer().min(1).max(100).default(20),
+  status: Joi.string()
+    .valid("pending", "processing", "completed", "failed", "rolled_back")
+    .optional(),
+  startDate: Joi.date().optional(),
+  endDate: Joi.date()
+    .when("startDate", {
+      is: Joi.exist(),
+      then: Joi.date().greater(Joi.ref("startDate")),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)